study-builder/tenants/register-user-in-pepper.js

```diff
@@ -20,7 +20,7 @@ function (user, context, callback) {
     var pepperClientClaim = 'https://datadonationplatform.org/cid';
     var pepperTenantClaim = 'https://datadonationplatform.org/t';
 
-    var tenantDomain = 'https://' + context.tenant + '.auth0.com/';
+    var tenantDomain = 'https://' + auth0.domain + '/';
```

Comment:
`auth0` is a built-in variable provided by Auth0. It is an instance of the Management API library. It looks like the following. We can use the `auth0.domain` since all the clients in a tenant should have the same value here. Docs: https://auth0.com/docs/rules/use-management-api.

<img width="483" alt="Screen Shot 2020-08-19 at 4 34 10 PM" src="https://user-images.githubusercontent.com/5713833/90687962-b40e3800-e23b-11ea-9db6-0d522f4fb6a2.png">